### PR TITLE
Fix/earn listing

### PR DESCRIPTION
**Pull request by sk-enya**
:clipboard:  closes: 

## Overview

Describe what your Pull Request is about in a few sentences.

## Tasks

- [x] No debug console statement added 
- [x] Should use only typescript.
- [x] Does have unit test for each touched file with 90% coverage
- [ ] Does have an integration test if any UI changes.

## Changes

Describe your changes and implementation choices

- Cleanup Earn Component 
- Prepare new Earnlisting container component 
- Prepare new EarnListItem component with functional component for react and batter handling of state.
- Rarn listing of stake and rewards
- Refactored functional logic for unstake / withdrawal

## Testing

Snapshots for test coverage for changed files.

## UI Snaps

Add screenshots or video if any visuals changes are there so would be helpful for review.

### CODE REVIEW

File: src/containers/earn/Earn.tsx
Comment (jemeza) [inline src/containers/earn/Earn.tsx:1]: There is an error in this file, I believe that's why the end to end test are not passing.

File: src/containers/earn/EarnList/earnListItem/index.tsx [hunk 1]
```diff
@@ -0,0 +1,269 @@
...
+  selectAccountEnabled,
+  selectlayer1Balance,
+  selectlayer2Balance,
+} from 'selectors'
+import networkService from 'services/networkService'
+import { LiquidityPoolLayer } from 'types/earn.types'
+import { formatLargeNumber, logAmount, powAmount } from 'util/amountConvert'
+import { EarnListItemContainer, IconWrapper, MoreActionIcon } from './styles'
+
+interface EarnListItemProps {
```
Comment (jemeza) [inline src/containers/earn/EarnList/earnListItem/index.tsx:22]: Would it be appropriate to put these in a types file?

File: src/reducers/earnReducer.ts [hunk 2]
```diff
@@ -23,10 +23,10 @@ if (networkService?.addresses) {
 const initialState = {
   poolInfo: {
     L1LP: {
-      [allAddresses.L1_ETH_Address]: {},
```
Comment (jemeza) [inline src/reducers/earnReducer.ts:26]: If the line will be commented out can we just delete it?

**Review by jemeza — CHANGES_REQUESTED** [review]
Looking good, let's just get the test passing, left some other feedback.

File: src/containers/earn/Earn.tsx
Comment (sk-enya) [reply src/containers/earn/Earn.tsx:1]: let me take look

**Pull request closed without merging**